graphics/diff-pdf/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                diff-pdf
+version             0.3
+categories          graphics
+maintainers         {@pietvo vanoostrum.org:piet}
+platforms           darwin
+supported_archs     i386 x86_64
+license             GPL-2
+
+description         diff-pdf is a tool for visually comparing two PDFs.
+long_description \
+    It takes two PDF files as arguments. By default, its only output is \
+    its return code, which is 0 if there are no differences and 1 if the \
+    two PDFs differ. If given the `--output-diff` option, it produces a \
+    PDF file with visually highlighted differences. \
+    With the `--view` argument it opens a window that lets you view the \
+    files' pages and zoom in on details.
+       
+homepage            http://vslavik.github.io/diff-pdf
+
+PortGroup           github 1.0
+github.setup        vslavik diff-pdf 0.3 v
+github.tarball_from releases
```

Comment:
These 3 lines should be moved to the top of the Portfile just below the `PortSystem` line. Then the version line will be redundant and can be removed.

Reply:
OK, I'll do that.
BTW, I suppose "name   diff-pdf" is still required.

Reply:
Good point. That line is also redundant and can be removed.